values.yaml

```diff
@@ -1004,6 +1004,59 @@ meshGateway:
   #     "annotation-key": "annotation-value"
   annotations: null
 
+ingressGateways:
+  enabled: true
+
+  # Defaults sets default values for all gateway fields. With the exception
+  # of annotations, defining any of these values in the `gateways` list
+  # will override the default values provided here.
+  defaults:
+    replicas: 2
+    imageEnvoy: "envoyproxy/envoy:v1.13.0"
```

Comment:
Can we pull this up to a global imageEnvoy so it's not duplicated across all 3 gateways?

Reply:
Connect also uses an `imageEnvoy` field. Do we want to consolidate all of them? Or leave the option to override that one?

Reply:
yeah not 100% sure without looking through all the implications so this might not work but my thought was add `global.imageEnvoy` and then for where we were previously using imageEnvoy keep that but use global as the default and for new locations don't offer a component-specific imageEnvoy.